packages/mobile/src/sentry/saga.ts

```diff
@@ -0,0 +1,11 @@
+import { spawn, takeEvery } from 'redux-saga/effects'
+import { Actions } from 'src/sentry/actions'
+import { initializeSentryUserContext } from 'src/sentry/Sentry'
+
+export function* watchLanguage() {
+  yield takeEvery(Actions.INITIALIZE_SENTRY_USER_CONTEXT, initializeSentryUserContext)
```

Comment:
This could probably go directly in `sentrySaga` we're unlikely to have other sentry related sagas. It's nbd but this will spawn the watcher which itself forks a new loop.